azext_iot/digitaltwins/_help.py

```diff
@@ -540,6 +540,16 @@ def load_digitaltwins_help():
             az dt twin delete -n {instance_or_hostname} --twin-id {twin_id} --etag {etag}
     """
 
+    helps["dt twin delete-all"] = """
+        type: command
+        short-summary: Deletes all digital twins within the specified resource, including all relationships for those twins.
```

Comment:
Instead of saying "within the specified resource", I think the description used in relationship delete-all is a bit more clear where it says "... on/within a Digital Twins instance,"